### ISSUE TITLE
More then one column in select statement

### ISSUE THREAD

**Issue by KrzysztofI** [apsmir/custom_field_sql#2]
Hi, In the case when sql query have 2 or more columns field have just first column (ok), but it does not work correctly - the value change (especially when second column in few rows are null).
As i wrote at redmine.org - useful plugin.
Regards,
KI

**Comment by apsmir** [comment]
Only the first column is used in selection. If you need to display two or more columns, use concatenation

**Comment by KrzysztofI** [comment]
Yes, maybe it's good to display some message about one column statement.

**Comment by konpega** [comment]
At first, this is wonderfull.

Question is related this subject but a little bit different. I am new to Redmine. In access we may see the columns we refer more than one. And with this opportunity / information you may easily select between choices (You may not be able to memorize all numbers). Think that Col A : Code, Col B : Explanation. We will see both columns, when we select only the the code should be written to the desired cell.

Thanks in advance
